src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs

```diff
@@ -193,15 +191,7 @@ protected override void Load(ContainerBuilder builder)
                     .ListeningCommands(typeof(CreateOffchainCashoutRequestCommand), typeof(CreateOffchainCashinRequestCommand))
                         .On(defaultRoute)
                     .WithCommandsHandler<OffchainCommandHandler>(),
-
-                Register.BoundedContext(BoundedContexts.Solarcoin)
-                    .FailedCommandRetryDelay(defaultRetryDelay)
-                    .ListeningCommands(typeof(SolarCashOutCommand))
-                        .On(defaultRoute)
-                    .PublishingEvents(typeof(SolarCashOutCompletedEvent))
```

Comment:
SolarCashOutCompletedEvent class is not deleted. Is it ok?

Reply:
done